bin/unix/install-dependencies/debian-ubuntu.sh

```diff
@@ -0,0 +1,22 @@
+#/bin/bash
+#
+## 
+## Usage: debian-ubuntu
+## 
+## This script automates the building of ownCloud documentation on Debian and Ubuntu distributions.
+## While new, it should handle installing all of the required platform depencies, and afterwards
+## build the documentation. 
+##
+## Author: Matthew Setter <msetter@owncloud.com>
+##
+
+function install_dependencies()
+{
+    # Update the Apt cache so that there are no errors based on an outdated package database
+    sudo apt-get update;
```

Comment:
Terminating lines with a semicolon is redundant in a bash shell script and just clutters the code.